wget : add some options (#2798)
tldr-pages/tldr: pages/common/wget.md

`wget -O {{bar}} {{https://example.com/foo}}`

- Download a single web page and all its resources with 3-second intervals between requests (scripts, stylesheets, images, etc.):

> `wget --page-requisites --convert-links --wait=3 {{https://example.com/somepage.html}}`

Reviewer: I don't know how much sense this example makes. Who would want to wait 3 seconds between each request when dumping a web page and its resources?

- Download all listed files within a directory and its sub-directories (does not download embedded page elements):

`wget --mirror --no-parent {{https://example.com/somepath/}}`

> - Limit the download speed with specific number of retry:

Reviewer: ```suggestion
- Limit the download speed and the number of retries:
```

> `wget --limit-rate={{300k}} -tries={{100}} {{https://example.com/somepath/}}`

Reviewer: ```suggestion
`wget --limit-rate={{300k}} --tries={{100}} {{https://example.com/somepath/}}`
```
Typo.

> - Download a file from a password protected site and FTP:

Reviewer: This does not make much sense. FTP and HTTP Basic Auth are two different things and have different command-line options for user and password. You should pick one between the two (I'd suggest keeping FTP as it is more common).

Author: Specify the user and assword work for both FTP and HTTP file. These parameters can be overridden using the --ftp-user and --ftp-password options for FTP connections and the --http-user and --http-password options for HTTP connections.

Reviewer: Oh, if those parameters are the same then it's cool. Should probably reword the example description though. Check my new comment below.

> - Download a file from a password protected site and FTP:

Reviewer: ```suggestion
- Download a file from an HTTP server using Basic Auth (also works for FTP):
```
Sounds better, what do you think?

Author: Yes :+1: ,much better.
thanks

`wget --user={{username}} --password={{password}} {{https://example.com}}`

- Continue an incomplete download:

`wget -c {{https://example.com}}`

> - Download all URLs stored in a text file to specific directory:

Reviewer: ```suggestion
- Download all URLs stored in a text file to a specific directory:
```
Typo.
